### PR TITLE
feature/TMVP-00/Initial version release

### PR DESCRIPTION
**Pull request by joel-wright**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+[tool.poetry]
+name = "di-service-locator"
+version = "0"
+description = "Injection utilities to obtain instances of dynamically created features from config"
+authors = []
+license = "MIT"
+readme = "README.md"
```
Comment (rafaelcpalmeida) [inline pyproject.toml:7]:
```suggestion
readme = "README.md"
homepage = "https://deeperinsights.com"
repository = "https://github.com/skimit/di-logging"
```

Comment (joel-wright) [reply pyproject.toml:7]: I'll just add a commit manually

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+homepage = "https://deeperinsights.com"
+repository = "https://github.com/skimit/di-logging"
```
Comment (rafaelcpalmeida) [inline pyproject.toml:9]:
```suggestion
repository = "https://github.com/skimit/di-service-locator"
```

File: make.cmd [hunk 3]
```diff
@@ -0,0 +1 @@
+@wsl WIN_BIN=y make %1
```
Comment (rafaelcpalmeida) [inline make.cmd:1]:
```suggestion
@wsl WIN_BIN=y make %1

```

File: docker/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+version: "3.9"
+services:
+  web:
+    container_name: ${PROJECT_NAME}
+    build:
+      context: ..
+      dockerfile: docker/Dev.Dockerfile
+    env_file: ../.env
+    environment:
```
Comment (psi77) [inline docker/docker-compose.yml:9]: we can get rid of this because we've got rid of the job queues

File: docker/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      - SQS_ENDPOINT_URL=http://${PROJECT_NAME}-elasticmq:9324
+    volumes:
+      - ..:/usr/src/app
+      - venvs:/venvs
+      - app_volume:/usr/src/circleci
+    depends_on:
+      - elasticmq
+
+  elasticmq:
```
Comment (psi77) [inline docker/docker-compose.yml:18]: Don't need this because we've removed job queues

File: pyproject.toml [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+include = [
+    { path = "LICENSE", format = ["sdist", "wheel"] },
+    { path = "README.md", format = "sdist" },
+    { path = "tests", format = "sdist" }
+]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+di-logging = "^0.1.0"
```
Comment (psi77) [inline pyproject.toml:18]: woohoo!

File: tests/test_features.json [hunk 7]
```diff
@@ -0,0 +1,9 @@
+{
+    "version": 1,
+    "features": {
+        "instrument": {
+            "factory": "di_features.feature_defs.instrumentation.BasicLogInstrument",
```
Comment (psi77) [inline tests/test_features.json:5]: These are the old package names

File: docker/docker-compose.yml (hunk 4 above)
Comment (joel-wright) [reply docker/docker-compose.yml:9]: good point

Comment (joel-wright) [reply docker/docker-compose.yml:9]: done

File: docker/docker-compose.yml (hunk 5 above)
Comment (joel-wright) [reply docker/docker-compose.yml:18]: done

**Review by psi77 — APPROVED** [review]
Looks good!

**Review by rafaelcpalmeida — DISMISSED** [review]

**Review by rafaelcpalmeida — APPROVED** [review]
LGTM

**Review by psi77 — DISMISSED** [review]

**Review by rafaelcpalmeida — APPROVED** [review]

**Pull request merged**